src/components/stepforms/widgets/Multiselect.vue

```diff
@@ -124,6 +121,14 @@ export default class MultiselectWidget extends Mixins(FormWidget) {
     const identifier = extractVariableIdentifier(value, this.variableDelimiters);
     return identifier != null;
   }
+
+  customLabel(option) {
+    if (typeof option === 'object' && this.label) {
```

Comment:
Maybe it would be better to test for `this.label != null`, just in case the property is one day `0` or something that evaluate falsy.

Reply:
Done